tests/suites/test_suite_ecp.data

```diff
@@ -237,9 +237,29 @@ ECP read binary #4 (non-zero, invalid ilen)
 depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
 ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"04001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
+ECP read binary #4a (non-zero, invalid ilen)
+depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"03001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
```

Comment:
Ok, that's that case where ilen is too small. Let's also have a case where it's too big.

I acknowledge that the uncompressed case has the same issue, but we are [in the process of improving our testing](https://github.com/orgs/Mbed-TLS/projects/1#column-18776795) so let's try to get it right from the start for new features.

Reply:
:rocket: